internal/mage/sdk/elixir.go

```diff
@@ -142,7 +143,45 @@ func (Elixir) Generate(ctx context.Context) error {
 
 // Publish publishes the Elixir SDK
 func (Elixir) Publish(ctx context.Context, tag string) error {
-	return nil
+	c, err := dagger.Connect(ctx, dagger.WithLogOutput(os.Stderr))
+	if err != nil {
+		return err
+	}
+	defer c.Close()
+
+	var (
+		version     = strings.TrimPrefix(tag, "sdk/elixir/v")
+		versionFile = "sdk/elixir/VERSION"
+		hexAPIKey   = os.Getenv("HEX_API_KEY")
+		dryRun      = os.Getenv("HEX_DRY_RUN")
+	)
+
+	if hexAPIKey == "" {
+		return errors.New("HEX_API_KEY environment variable must be set")
+	}
+
+	if err := os.WriteFile(versionFile, []byte(version), 0o600); err != nil {
+		return err
+	}
+	defer func() {
+		// Ensure to not make version file dirty.
+		os.WriteFile(versionFile, []byte("0.0.0\n"), 0o600)
+	}()
+
+	args := []string{"mix", "hex.publish", "--yes"}
+	if dryRun == "1" {
+		args = append(args, "--dry-run")
+	}
+
+	// TODO: copy LICENSE?
```

Comment:
I not what to do with LICENSE file. But I can copying LICENSE from root repository into sdk/elixir before publish and then deleting it after every operation is done (defer).

Reply:
Done!